core-rust/state-manager/src/jni/test_state_reader.rs

```diff
@@ -202,7 +200,7 @@ extern "system" fn Java_com_radixdlt_testutil_TestStateReader_componentXrdAmount
             let read_store = database.read();
 
             // a quick fix for handling virtual accounts
-            // TODO: fix upstream
+            // TODO(during review): has this been "fixed upstream"?
```

Comment:
No, it hasn't - but I suggest removing this TODO. I believe this is only used in tests anyway.